models/tanh_rate.h

```diff
@@ -0,0 +1,117 @@
+/*
+ *  tanh_rate.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef TANH_RATE_H
+#define TANH_RATE_H
+
+// Includes from models:
+#include "rate_neuron_ipn.h"
+#include "rate_neuron_ipn_impl.h"
+#include "rate_neuron_opn.h"
+#include "rate_neuron_opn_impl.h"
+
+
+namespace nest
+{
+/* BeginDocumentation
+Name: tanh_rate - rate model with hyperbolic tangent non-linearity
+
+Description:
+
+ tanh_rate is an implementation of a non-linear rate model with either
+ input (tanh_rate_ipn) or output noise (tanh_rate_opn) and gain function
+ Phi(h) = tanh(g * h) and Psi(h) = h for linear_summation = True
```

Comment:
Why can the user only choose the gain, but not the location of the inflection point (tanh(gh-x))?.